Pyspark/regular_patients/main.py

```diff
@@ -0,0 +1,45 @@
+'''This module implements the PatientsData class to analyze the regular patient data'''
+from regular_patients.logger import Log4J
+from regular_patients import PatientsData
+
+if __name__ == '__main__':
+
+    #solo para probar
+    import os
+    print("Current Working Directory:", os.getcwd())
+
+    FILE_PATH = "file:///Pyspark/regular_patients/input/enroll.csv"
+    #creating the spark session and loading the data
+    patients = PatientsData(FILE_PATH)
+
+    #instantiating my logger
+    logger = Log4J(patients.spark)
+    logger.info('Starting PatientsData')
+
+    #tranforming te effective_from_date field to date type
+    patients.format_date()
+
+    #filtering by date
+    END_DATE = '2016-09-30'
+    START_DATE = '2015-10-01'
+    patients.filter_by_date_range(START_DATE, END_DATE)
+    df_patients = patients.df_patients
+    df_patients.printSchema()
+    df_patients.show(10)
+
+    #identifying consecutives
+    consecutives = patients.checking_consecutives_records()
+    consecutives.show(10)
+
+    #getting the continuous records for at least 5, 9, or 11 months Dataframe
+    df_patients = patients.get_result()
+
+    #generating the output file
+    OUTPUT_PATH = "file:///Pyspark/regular_patients/output"
+    df_patients.write.mode("overwrite").csv(OUTPUT_PATH)
+    df_patients.show(10)
+    df_patients.printSchema()
```

Comment:
this code can also be moved to Patients data class as it is part of pipeline, you can craete a separate method from it